templates/service/project/AppDependencies.scala

```diff
@@ -12,7 +12,7 @@ object AppDependencies {
     "uk.gov.hmrc"             %% "play-ui"                  % "$!playUiVersion!$",
     <!--(end)-->
     <!--(if MONGO)-->
-    "uk.gov.hmrc"             %% "play-reactivemongo"       % "$!playReactivemongoVersion!$",
+    "uk.gov.hmrc"             %% "simple-reactivemongo"       % "$!simpleReactivemongoVersion!$",
```

Comment:
percent signs are no longer vertically aligned

Reply:
Fixed in https://github.com/hmrc/init-service/commit/547e9cdd38344cccbb1e904429264575a1a86eca

Reply:
thanks!